.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+name: CI Build
```

Comment:
should we not remove .travis.yaml when we migrate from travis to githubactions?

Reply:
That is not required to be done for getting started with github actions, yes but we should keep it in action item.